packages/client/src/__mocks__/addressMockNoZoningplans.ts

```diff
@@ -11,5 +13,5 @@ export default {
     },
   ],
   streetName: "streetname",
-  zoningPlans: [],
-};
+  zoningPlans: [{}],
```

Comment:
`ZoningPlan.name` shouldn't be optional (according to GQL schema). Can we make it non-optional op `type ZoningPlan` in `../types/`?

`zoningPlans: [{}],` should be replaced with an actual zoning plan object which includes the name. Or remove the `{}`.

Reply:
Ah okay, I thought it was optional. I will change this

Reply:
It's changed